windows/RNSVG/RenderableView.h

```diff
@@ -33,22 +35,18 @@ struct RenderableView : RenderableViewT<RenderableView> {
   float StrokeDashOffset() { return m_strokeDashOffset; }
   RNSVG::SVGLength StrokeWidth() { return m_strokeWidth; }
   Windows::Foundation::Collections::IVector<RNSVG::SVGLength> StrokeDashArray() { return m_strokeDashArray; }
-  Microsoft::Graphics::Canvas::Geometry::CanvasCapStyle StrokeLineCap() { return m_strokeLineCap; }
-  Microsoft::Graphics::Canvas::Geometry::CanvasLineJoin StrokeLineJoin() { return m_strokeLineJoin; }
-  Microsoft::Graphics::Canvas::Geometry::CanvasFilledRegionDetermination FillRule() { return m_fillRule; }
-  Microsoft::Graphics::Canvas::Geometry::CanvasGeometry ClipPathGeometry();
+  int32_t StrokeLineCap() { return m_strokeLineCap; }
+  int32_t StrokeLineJoin() { return m_strokeLineJoin; }
+  int32_t FillRule() { return m_fillRule; }
```

Comment:
What was the reason these should be ints here instead of their mapped enum values?